### PR TITLE
[Tracing][PHP] Add annotations doc

### PR DESCRIPTION
**Pull request by pierotibou**
### What does this PR do?
Add annotations to the custom instrumentation doc in Tracing / PHP

### Motivation
Feature has just been released.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by bwoebi — APPROVED** [review]

File: content/en/tracing/trace_collection/custom_instrumentation/php.md [hunk 1]
```diff
@@ -26,6 +26,33 @@ further_reading:
 If you have not yet read the instructions for auto-instrumentation and setup, start with the <a href="/tracing/setup/php/">PHP Setup Instructions</a>. Even if Datadog does not officially support your web framework, you may not need to perform any manual instrumentation. See <a href="/tracing/setup/php/#automatic-instrumentation">automatic instrumentation</a> for more details.
 </div>
 
+## Annotations
+
+If you run on PHP 8, starting v0.84 of the tracer, you can add attributes to your code to instrument it. It is a lighter alternative to [custom instrumentation written in code](#writing-custom-instrumentation). For instance, just add `#[DDTrace\Trace]` attribute to methods for Datadog to trace them.
```
Comment (jtappa) [inline content/en/tracing/trace_collection/custom_instrumentation/php.md:31]:
```suggestion
If you are using PHP 8, as of v0.84 of the tracer, you can add attributes to your code to instrument it. It is a lighter alternative to [custom instrumentation written in code](#writing-custom-instrumentation). For example, add the `#[DDTrace\Trace]` attribute to methods for Datadog to trace them.
```

File: content/en/tracing/trace_collection/custom_instrumentation/php.md [hunk 2]
```diff
@@ -26,6 +26,33 @@ further_reading:
...
+
+```php
+<?php
+class Foo {
+    #[DDTrace\Trace(name: "simplename", resource: "rsrc", type: "typeee", service: "test", tags: ["a" => "b"])]
```
Comment (jtappa) [inline content/en/tracing/trace_collection/custom_instrumentation/php.md:36]: is "typeee" an actual type...? if not can we make the example a little more realistic?

File: content/en/tracing/trace_collection/custom_instrumentation/php.md [hunk 3]
```diff
@@ -26,6 +26,33 @@ further_reading:
...
+    static function simple($arg) {}
+
+    #[DDTrace\Trace]
+    function bar() {
+      Foo::simple(1);
```
Comment (jtappa) [inline content/en/tracing/trace_collection/custom_instrumentation/php.md:41]: can we avoid foobars here and put in something more realistic for our readers?

**Review by jtappa — CHANGES_REQUESTED** [review]

File: content/en/tracing/trace_collection/custom_instrumentation/php.md (hunk 2 above)
Comment (pierotibou) [reply content/en/tracing/trace_collection/custom_instrumentation/php.md]: done in 19366c5d4ecfc72675ee41b6454fd20bd0f8f274

File: content/en/tracing/trace_collection/custom_instrumentation/php.md (hunk 3 above)
Comment (pierotibou) [reply content/en/tracing/trace_collection/custom_instrumentation/php.md]: done in 19366c5d4ecfc72675ee41b6454fd20bd0f8f274

**Review by jtappa — APPROVED** [review]

**Pull request merged**